types/project/CartConfigurationDraft.raml

```diff
@@ -0,0 +1,7 @@
+#%RAML 1.0 DataType
```

Comment:
I think that this file isn't required, because we don't have an update action that uses it. I would suggest to remove it.